global/js/main.js

```diff
@@ -777,7 +777,8 @@ var groupUtils = {
     },
     badgeHtmlFromGroupId: function (id, size = "small") {
         var group = this.getGroupFromId(id);
-        return `<a href="/misc/groups/?group=${id}"><div class="osekai__group-badge osekai__group-badge-${size}" style="--colour: ${group['Colour']}">${group['ShortName']}</div></a>`;
+        // this is technically illegal according to html spec but i don't care

```

Comment:
you don't but i do.
this is what the changes look like:
![image](https://user-images.githubusercontent.com/30161277/207598277-fbc8ca6c-6416-47b4-aca8-707c155176dc.png)
and this is without object:
![image](https://user-images.githubusercontent.com/30161277/207598310-82ce9c0b-d435-4083-bb38-247e3d9e9252.png)
get rid of the object and everything should be fine

Reply:
object breaks everything More. having an <a> tag inside an <a> tag causes the issue we had in the first place. removing it will literally break everything more so.

Reply:
![image](https://user-images.githubusercontent.com/33783503/207599024-36105b84-914b-4ff2-918e-6f9147ad4e31.png)
this also isn't even an issue when using object on my side. tested on both chrome and firefox. if you can tell me how to reproduce that issue i'll try fix it but it seems fine to me

Reply:
happy to follow up with more info on discord if wanted

Reply:
but it looks good :<

Reply:
> in that instance, I would use a simple ul / li treatment instead of an <object> list

**you can't put an a tag inside another a tag this is what this is to fix because putting it inside an object fixes it**